application/modules/home/controllers/Home.php

```diff
@@ -43,4 +43,17 @@ public function index() {
 		// load view
 		$this->load_view('top', $this->data);
 	}
+
+	/**
+	 * Test side ci
+	 *
+	 * @return void
+	 */
+	public function side_ci()
+	{
+		$this->data["exhibition"]["name"] = "test";
```

Comment:
__[code_sniffer]__
* Single-quoted strings should be used unless it contains variables, special chars like \n or single quotes.

<!-- Comment by [SideCI](https://sideci.com) -->